### PR TITLE
refactor: remove redundant code

### PR DESCRIPTION
**Pull request by mrazauskas**
It looks like by fixing typings some redundant code was added.

Might be I misunderstood something. On surface it looks like tests cannot reach these lines. So why to ship them to `npm`, if they do nothing?

### CODE REVIEW

**Comment by mrazauskas** [comment]
Is this simply a bug in Biome? Oh well..

**Comment by wellwelwel** [comment]
> Is this simply a bug in Biome? Oh well..

You're right. Not a bug, just a special rule for this file:

```jsonc
    {
      "include": ["src/modules/helpers/modifiers.ts"],
      "linter": {
        "rules": {
          "suspicious": {
            "useAwait": "off"
          }
        }
      }
    }
```

**Comment by mrazauskas** [comment]
Normally I think: if TypeScript does not complain, all is correct. So for now I used `// biome-ignore`, but I have a better idea. Working on that.

**Comment by mrazauskas** [comment]
Nope. That idea did not work. So I replaced the ignore comments with config override.

What you think?

File: src/modules/helpers/modifiers.ts [hunk 1]
```diff
@@ -5,58 +5,32 @@ import { format } from '../../services/format.js';
...
+export function skip(message: string, cb: () => unknown): void;
+export async function skip(cb: () => Promise<unknown>): Promise<void>;
+export function skip(cb: () => unknown): void;
 export async function skip(
   messageOrCb: string | (() => unknown) | (() => Promise<unknown>),
   _cb?: (() => unknown) | (() => Promise<unknown>)
 ): Promise<void> {
-  const message =
-    (typeof messageOrCb === 'string' && messageOrCb) || 'Skipping';
+  const message = typeof messageOrCb === 'string' ? messageOrCb : 'Skipping';
```
Comment (mrazauskas) [inline src/modules/helpers/modifiers.ts:31]: This change is not related. Just a simplification.

Comment (wellwelwel) [reply src/modules/helpers/modifiers.ts:31]: This logic is actually intended to prevent the user from passing an empty string. But I think that if it's the user's intention, I shouldn't interfere. LGTM.

**Comment by wellwelwel** [comment]
> What you think?

I didn't like the workaround I'd made, but I hadn't realized it was just a **Biome** rule. Thanks @mrazauskas 🤝✨
